<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="2" failed="1" ignored="8" total="13" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Regression Suite" duration-ms="77194" started-at="2021-11-15T15:27:35Z" finished-at="2021-11-15T15:28:52Z">
    <groups>
    </groups>
    <test name="Fund Transfer" duration-ms="77194" started-at="2021-11-15T15:27:35Z" finished-at="2021-11-15T15:28:52Z">
      <class name="testCases.LoginWithRegisteredUser">
        <test-method status="PASS" signature="initialization()[pri:0, instance:testCases.LoginWithRegisteredUser@61832929]" name="initialization" is-config="true" duration-ms="620" started-at="2021-11-15T20:57:34Z" finished-at="2021-11-15T20:57:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialization -->
        <test-method status="PASS" signature="isSkip()[pri:0, instance:testCases.LoginWithRegisteredUser@61832929]" name="isSkip" is-config="true" duration-ms="0" started-at="2021-11-15T20:57:35Z" finished-at="2021-11-15T20:57:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- isSkip -->
        <test-method status="PASS" signature="loginWithRegisteredUser(java.util.Hashtable)[pri:0, instance:testCases.LoginWithRegisteredUser@61832929]" name="loginWithRegisteredUser" duration-ms="21061" started-at="2021-11-15T20:57:35Z" data-provider="data_Collection" finished-at="2021-11-15T20:57:56Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Page_Title=My Store, Contact_No=533535353, Email_ID=testShyama@gmail.com, Password=passd1237, Sr.No=1.0, City=Mumbai, First_Name=Shyama, Last_Name=XYZ1}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginWithRegisteredUser -->
        <test-method status="PASS" signature="generateReport()[pri:0, instance:testCases.LoginWithRegisteredUser@61832929]" name="generateReport" is-config="true" duration-ms="271" started-at="2021-11-15T20:58:52Z" finished-at="2021-11-15T20:58:52Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generateReport -->
      </class> <!-- testCases.LoginWithRegisteredUser -->
      <class name="testCases.CreateAnAccountToMyStore4">
        <test-method status="PASS" signature="isSkip()[pri:0, instance:testCases.CreateAnAccountToMyStore4@26653222]" name="isSkip" is-config="true" duration-ms="1" started-at="2021-11-15T20:58:24Z" finished-at="2021-11-15T20:58:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- isSkip -->
        <test-method status="FAIL" signature="createAnAccountToMyStore4(java.util.Hashtable)[pri:0, instance:testCases.CreateAnAccountToMyStore4@26653222]" name="createAnAccountToMyStore4" duration-ms="26798" started-at="2021-11-15T20:58:24Z" data-provider="data_Collection" finished-at="2021-11-15T20:58:51Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Page_Title=My Store, Contact_No=1234567, Email_ID=testRahul354345643@gmail.com, Password=passd1234, Sr.No=1, City=Pune, First_Name=Rahul Ganjare, Last_Name=sdfsdf}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [My Store] but found [Login - My Store]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [My Store] but found [Login - My Store]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at utility.CommonMethods.verifyPageTitle(CommonMethods.java:42)
	at pages.AuthenticationPage.loginWithAlreadyRegistredUser(AuthenticationPage.java:24)
	at testCases.CreateAnAccountToMyStore4.createAnAccountToMyStore4(CreateAnAccountToMyStore4.java:52)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createAnAccountToMyStore4 -->
      </class> <!-- testCases.CreateAnAccountToMyStore4 -->
      <class name="testCases.CreateAnAccountToMyStore3">
        <test-method status="SKIP" signature="isSkip()[pri:0, instance:testCases.CreateAnAccountToMyStore3@5e5792a0]" name="isSkip" is-config="true" duration-ms="0" started-at="2021-11-15T20:58:24Z" finished-at="2021-11-15T20:58:24Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping Test case CreateAnAccountToMyStore3 as it's run mode is set to No]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping Test case CreateAnAccountToMyStore3 as it's run mode is set to No
	at testCases.CreateAnAccountToMyStore3.isSkip(CreateAnAccountToMyStore3.java:24)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:169)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- isSkip -->
        <test-method status="SKIP" signature="createAnAccountToMyStore3(java.util.Hashtable)[pri:0, instance:testCases.CreateAnAccountToMyStore3@5e5792a0]" name="createAnAccountToMyStore3" duration-ms="2" started-at="2021-11-15T20:58:24Z" data-provider="data_Collection" finished-at="2021-11-15T20:58:24Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Page_Title=My Store, Contact_No=678568624, Email_ID=testAnkita@gmail.com, Password=passd1238, Sr.No=1.0, City=Pune, First_Name=Ankita, Last_Name=XYZ2}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping Test case CreateAnAccountToMyStore3 as it's run mode is set to No]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping Test case CreateAnAccountToMyStore3 as it's run mode is set to No
	at testCases.CreateAnAccountToMyStore3.isSkip(CreateAnAccountToMyStore3.java:24)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:169)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createAnAccountToMyStore3 -->
      </class> <!-- testCases.CreateAnAccountToMyStore3 -->
      <class name="testCases.LoginWithRegisteredUser2">
        <test-method status="PASS" signature="isSkip()[pri:0, instance:testCases.LoginWithRegisteredUser2@29774679]" name="isSkip" is-config="true" duration-ms="0" started-at="2021-11-15T20:57:57Z" finished-at="2021-11-15T20:57:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- isSkip -->
        <test-method status="PASS" signature="loginWithRegisteredUser2(java.util.Hashtable)[pri:0, instance:testCases.LoginWithRegisteredUser2@29774679]" name="loginWithRegisteredUser2" duration-ms="27012" started-at="2021-11-15T20:57:57Z" data-provider="data_Collection" finished-at="2021-11-15T20:58:24Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Page_Title=My Store, Contact_No=678568624, Email_ID=testSagar@gmail.com, Password=passd1238, Sr.No=1.0, City=Pune, First_Name=Sagar, Last_Name=XYZ2}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginWithRegisteredUser2 -->
      </class> <!-- testCases.LoginWithRegisteredUser2 -->
      <class name="testCases.CreateAnAccountToMyStore2">
        <test-method status="SKIP" signature="isSkip()[pri:0, instance:testCases.CreateAnAccountToMyStore2@3ffc5af1]" name="isSkip" is-config="true" duration-ms="0" started-at="2021-11-15T20:58:24Z" finished-at="2021-11-15T20:58:24Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping Test case CreateAnAccountToMyStore2 as it's run mode is set to No]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping Test case CreateAnAccountToMyStore2 as it's run mode is set to No
	at testCases.CreateAnAccountToMyStore2.isSkip(CreateAnAccountToMyStore2.java:24)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:169)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- isSkip -->
        <test-method status="SKIP" signature="createAnAccountToMyStore2(java.util.Hashtable)[pri:0, instance:testCases.CreateAnAccountToMyStore2@3ffc5af1]" name="createAnAccountToMyStore2" duration-ms="1" started-at="2021-11-15T20:58:24Z" data-provider="data_Collection" finished-at="2021-11-15T20:58:24Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Page_Title=My Store, Contact_No=456464646, Email_ID=testArijit@gmail.com, Password=passd1236, Sr.No=1, City=Kolkata, First_Name=Arijit, Last_Name=XYZ}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping Test case CreateAnAccountToMyStore2 as it's run mode is set to No]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping Test case CreateAnAccountToMyStore2 as it's run mode is set to No
	at testCases.CreateAnAccountToMyStore2.isSkip(CreateAnAccountToMyStore2.java:24)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:169)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createAnAccountToMyStore2 -->
      </class> <!-- testCases.CreateAnAccountToMyStore2 -->
    </test> <!-- Fund Transfer -->
  </suite> <!-- Regression Suite -->
</testng-results>
